Temmietayo/All-Recipes
# #169997275 Template for User to search and also see popular recipes

#### What does this PR do?
Add the template for user to search recipe on home page .

#### Description of Task to be completed?
Create template page for user to search recipe

#### How should this be manually tested?

- clone the repo

- cd into the template directory

- open the home.html with any browser of your choice

- these template should open in that browser
#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[169997275](https://www.pivotaltracker.com/story/show/169997275)
